tutorials/best_practices/godot_interfaces.rst

```diff
@@ -46,18 +46,19 @@ access.
 
     # Note that users load scenes and scripts, by convention, with PascalCase
     # names (like typenames), often into constants.
-    const MyScene : = preload("my_scene.tscn") as PackedScene # Static load
-    const MyScript : = preload("my_script.gd") as Script
+    const MyScene := preload("my_scene.tscn") as PackedScene # Static load
+    const MyScript: Script = preload("my_script.gd")
 
     # This type's value varies, i.e. it is a variable, so it uses snake_case.
-    export(Script) var script_type: Script
+    @export var script_type: Script
 
     # If need an "export const var" (which doesn't exist), use a conditional
     # setter for a tool script that checks if it's executing in the editor.
-    tool # Must place at top of file.
+    @tool # Must place at top of file.
 
     # Must configure from the editor, defaults to null.
-    export(Script) var const_script setget set_const_script
+    @export var const_script: Script:
+        set = set_const_script
```

Comment:
```suggestion
    @export var const_script : Script:
        set = set_const_script
```
Looks all good except this one. This seems pretty hard to read to me, not sure how to improve it. And should there be a space before the type for consistency, like this?